tests/entry.rs

```diff
@@ -130,5 +180,8 @@ fn good_parent_paths_ok() {
 
     let td = t!(TempDir::new("tar"));
     t!(ar.unpack(td.path()));
-    t!(File::open(td.path().join("foo/bar")));
+    t!(td.path().join("foo").join("bar").read_link());
+    let dst = t!(td.path().join("foo").join("bar").canonicalize());
```

Comment:
Yes, and I'm puzzled. I expanded the previous test into path-joining, readlink(), and canonicalize() in order to understand what's going on, but the original one was also failing. It looks like windows doesn't like the `../bar` symlink. This crate is converting `\` to `/` in symlinks, so `..\bar` becomes `../bar`. This may be part of the issue, but it seems to be on purpose (and actually covered by unit testing). I'm not familiar with windows api, how do relative symlinks and separators work there?